services/api/scripts/entrypoints/cli.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 echo "" > /service/crontab.log
-./scripts/fixtures/load
```

Comment:
Can we make this an IF statement that avoids it for Production but still allows it for Staging

Reply:
Added an IF statement in the following PR: https://github.com/bedrockio/bedrock-core/pull/232